packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -190,6 +269,7 @@ export default compose( [
 		const selectedBlock = getSelectedBlock();
 		const parentId = getBlockRootClientId( clientId );
 		const parentBlock = getBlock( parentId );
+		const isGroupParent = parentBlock && parentBlock.name === 'core/group';
 
 		const isMediaText = selectedBlock && selectedBlock.name === 'core/media-text';
```

Comment:
is "isMediaText" still used?

Reply:
No, it was used to hide `FloatingToolbar` and I didn't catch that after turn it on. Thanks.

Reply:
fixed